### PR TITLE
Move DailyBuildNewFeatures, QuickStart, DownLoad & Installs 

### PR DESCRIPTION
**Pull request by hamishwillee**
This moves files into a more sensible structure and fixes up links
```
getting started
   quick_start.md
   downloads_and_installs.md
releases
  daily_build_new_features.md
  release_notes.md
```
All the files are redirected using the bulk_redirects plugin.

### CODE REVIEW

**Comment by hamishwillee** [comment]
@DonLakeFlyer This is quite a big structural move, hence the sanity check request for review. I have made sure that download_and_install is redirected!

If you are happy with this I propose that we next:
1. Split **download_and_install.md**, moving"Daily Builds" section into own doc in /releases (and linking to it from d- linking to it from **download_and_install.md**)
1. Change the structure so everything to do with releases is together
```
Quick Start
Downloads & Installs
...
Releases
  Using Daily Builds
  Daily Build New Features
  Release Notes
```
Basically I want to simplify the basic installation case and pull Daily builds and releases as a more complicated case. Does this make sense to you.

**Comment by DonLakeFlyer** [comment]
All makes sense to me
